### ISSUE TITLE
requirements.txt has some problems

### ISSUE THREAD

**Issue by lihua8848** [xiaogang00/MTFormer#1]
Thank you for your work.  When i execute `pip install -r requirements.txt`, there are some problems. I'm looking for your reply. Thank you for your work again.

**Comment by Klodivio355** [comment]
@lihua8848 Indeed. The following environment worked for me (yml file). 

name: py36
channels:
  - pytorch
  - nvidia
  - defaults
dependencies:
  - _libgcc_mutex=0.1=main
  - _openmp_mutex=5.1=1_gnu
  - blas=1.0=mkl
  - bzip2=1.0.8=h7b6447c_0
  - ca-certificates=2023.05.30=h06a4308_0
  - certifi=2021.5.30=py36h06a4308_0
  - cuda-cudart=11.7.99=0
  - cuda-cupti=11.7.101=0
  - cuda-libraries=11.7.1=0
  - cuda-nvrtc=11.7.99=0
  - cuda-nvtx=11.7.91=0
  - cuda-runtime=11.7.1=0
  - cudatoolkit=11.3.1=ha36c431_9
  - dataclasses=0.8=pyh4f3eec9_6
  - ffmpeg=4.3=hf484d3e_0
  - freetype=2.12.1=h4a9f257_0
  - gmp=6.2.1=h295c915_3
  - gnutls=3.6.15=he1e5248_0
  - intel-openmp=2022.1.0=h9e868ea_3769
  - jpeg=9e=h5eee18b_1
  - lame=3.100=h7b6447c_0
  - lcms2=2.12=h3be6417_0
  - ld_impl_linux-64=2.38=h1181459_1
  - lerc=3.0=h295c915_0
  - libcublas=11.10.3.66=0
  - libcufft=10.7.2.124=h4fbf590_0
  - libcufile=1.7.0.149=0
  - libcurand=10.3.3.53=0
  - libcusolver=11.4.0.1=0
  - libcusparse=11.7.4.91=0
  - libdeflate=1.17=h5eee18b_0
  - libffi=3.3=he6710b0_2
  - libgcc-ng=11.2.0=h1234567_1
  - libgomp=11.2.0=h1234567_1
  - libiconv=1.16=h7f8727e_2
  - libidn2=2.3.4=h5eee18b_0
  - libnpp=11.7.4.75=0
  - libnvjpeg=11.8.0.2=0
  - libpng=1.6.39=h5eee18b_0
  - libstdcxx-ng=11.2.0=h1234567_1
  - libtasn1=4.19.0=h5eee18b_0
  - libtiff=4.5.0=h6a678d5_2
  - libunistring=0.9.10=h27cfd23_0
  - libuv=1.44.2=h5eee18b_0
  - libwebp-base=1.2.4=h5eee18b_1
  - lz4-c=1.9.4=h6a678d5_0
  - mkl=2020.2=256
  - mkl-service=2.3.0=py36he8ac12f_0
  - mkl_fft=1.3.0=py36h54f3939_0
  - mkl_random=1.1.1=py36h0573a6f_0
  - ncurses=6.4=h6a678d5_0
  - nettle=3.7.3=hbbd107a_1
  - numpy-base=1.19.2=py36hfa32c7d_0
  - olefile=0.46=py36_0
  - openh264=2.1.1=h4ff587b_0
  - openjpeg=2.4.0=h3ad879b_0
  - openssl=1.1.1u=h7f8727e_0
  - pip=21.2.2=py36h06a4308_0
  - python=3.6.13=h12debd9_1
  - pytorch=1.10.2=py3.6_cuda11.3_cudnn8.2.0_0
  - pytorch-cuda=11.7=h778d358_5
  - pytorch-mutex=1.0=cuda
  - readline=8.2=h5eee18b_0
  - sqlite=3.41.2=h5eee18b_0
  - tk=8.6.12=h1ccaba5_0
  - torchaudio=0.10.2=py36_cu113
  - typing_extensions=4.1.1=pyh06a4308_0
  - wheel=0.37.1=pyhd3eb1b0_0
  - xz=5.4.2=h5eee18b_0
  - zlib=1.2.13=h5eee18b_0
  - zstd=1.5.5=hc292b87_0
  - pip:
      - argparse==1.1
      - asn1crypto==1.5.1
      - cffi==1.15.1
      - charset-normalizer==2.0.12
      - cryptography==2.2.2
      - cycler==0.11.0
      - decorator==5.1.1
      - easydict==1.10
      - einops==0.4.0
      - filelock==3.4.1
      - h5py==2.7.1
      - huggingface-hub==0.4.0
      - idna==3.4
      - imageio==2.3.0
      - importlib-metadata==4.8.3
      - kiwisolver==1.3.1
      - matplotlib==2.2.2
      - networkx==2.1
      - numpy==1.19.5
      - opencv-python==4.1.2.30
      - packaging==21.3
      - pillow==5.1.0
      - protobuf==3.19.6
      - pycparser==2.21
      - pyparsing==3.1.0
      - python-dateutil==2.8.2
      - pytz==2023.3
      - pywavelets==1.1.1
      - pyyaml==3.12
      - requests==2.26.0
      - scikit-image==0.17.2
      - scikit-learn==0.19.1
      - scipy==1.1.0
      - setuptools==57.4.0
      - six==1.11.0
      - tensorboardx==2.6
      - termcolor==1.1.0
      - tifffile==2020.9.3
      - timm==0.6.12
      - torch==1.8.1
      - torchvision==0.9.1
      - tqdm==4.62.3
      - urllib3==1.26.16
      - zipp==3.6.0
prefix: /home/maximefontana/anaconda3/envs/py36
